cognite/powerops/client/shop/data_classes/shop_case.py

```diff
@@ -55,30 +56,40 @@ def __init__(
     ) -> None:
         if data and file_path:
             raise ValueError("Cannot specify both data and file_path")
-        self.data = {}
-        self.excess_yaml_parts: list[str] = []
+
         if data:
-            self.load_case_data(data)
-        elif file_path:
-            self.load_case_file(file_path)
+            self._case_string = data.lstrip("\n")
+        else:
+            self._case_string = self.load_case_file(file_path)
```

Comment:
Should be the class name, not self since `load_case_file` is a static method

Reply:
Updated.

Reply:
Updated good spot